### PR TITLE
Switch snuggets to xlsx

### PR DESCRIPTION
**Pull request by eldang**
1. Switches both snuggets.csv and slideshow.csv to .xlsx formats, with `snugget_load.py` updated to process the Excel files.
2. Copies the `checkHTMLTagClosures` function that we'd added to the Seattle version of `snugget_load.py`, so whoever runs this script will now get warnings of unclosed HTML tags in columns of `snuggets.xlsx` that are either called 'text' or have a name beginning with 'text-'.

Note that by default the XLSX file reader assumes that it can rely on the active worksheet in a file being the right one to read.  This is only a safe assumption when files have only one worksheet.  The function can also be called with a worksheet name specified, if we decide to start using it that way.

### CODE REVIEW

**Comment by eldang** [comment]
I should have mentioned: this also replaces both the ZIP and uncompressed versions of the /data/ folder with some sample Portland-area data.

File: .gitignore [hunk 1]
```diff
@@ -84,3 +84,9 @@ myenv/
 node_modules/
 
 webpack-stats.json
+
+# Excel temp files
+~$*.xlsx
+
+# Auto-assembled media file
```
Comment (nein09) [inline .gitignore]: this is a folder, but i think that's fine

File: disasterinfosite/admin.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 # GENERATED CODE GOES HERE
 # DO NOT MANUALLY EDIT CODE IN THIS SECTION - IT WILL BE OVERWRITTEN
 # adminModelImports
-from .models import EmbedSnugget, TextSnugget, SnuggetSection, Location, SiteSettings, SupplyKit, ImportantLink
+from .models import EmbedSnugget, TextSnugget, SnuggetSection, Location, SiteSettings, SupplyKit, ImportantLink, RDPOLiquefact_Clark, RDPO_counties, RDPOLiquefaction_OR, RDPOCascadiaM9_3_Clark, RDPO_region, RDPOCascadiaM9_OR
```
Comment (nein09) [inline disasterinfosite/admin.py]: Can we leave the stuff that's specific to the sample data out of these files (e.g. have something checked into the repo that is as if we have never run `import.py`)?

Comment (nein09) [reply disasterinfosite/admin.py]: I'm glad to have the sample data in there but I think it'll be easier to create new projects from this repo if we don't have the generated code stuff in here.

**Comment by nein09** [comment]
Wait, I don't think we want the uncompressed data folder, just the zip.

File: .gitignore (hunk 1 above)
Comment (eldang) [reply .gitignore]: Um, right you are.  I might as well update the comment while I'm making other commits.

File: disasterinfosite/admin.py (hunk 2 above)
Comment (eldang) [reply disasterinfosite/admin.py]: Oooooohhhhhh, I'm out of practice with this project....  I'll fix that this afternoon and make a fresh commit.

**Comment by eldang** [comment]
OK, I'll take the uncompressed folder out and add it to gitignore.  That way we also won't have such huge changelists.

File: disasterinfosite/admin.py (hunk 2 above)
Comment (nein09) [reply disasterinfosite/admin.py]: I know that feeling!

**Comment by nein09** [comment]
Awesome, thank you- that will help. This repo *should* have git-lfs enabled as far as I know.

**Comment by eldang** [comment]
> Wait, I don't think we want the uncompressed data folder, just the zip.

LOL, that was already in gitignore but apparently a leading space was stopping it from being processed correctly.

**Comment by nein09** [comment]
ooooops.

**Comment by eldang** [comment]
OK, I think this is ready to merge now.  The changelist is certainly easier to read without all those other files in it, but one thing to note is that snuggets.xlsx and slideshow.xlsx are now only in Github _within_ the `data.zip` archive.  So I wonder if we should put a copy of those somewhere else so they're easier to find?

**Comment by nein09** [comment]
Or maybe just an example of the headers and a couple of rows for each in the readme?

**Pull request merged**